var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -119,12 +142,7 @@ def install(self, spec, prefix):
     def setup_environment(self, spack_env, run_env):
         SimModel.setup_environment(self, spack_env, run_env)
         run_env.set('HOC_LIBRARY_PATH', self.prefix.lib.hoc)
-        run_env.set('NEURON_INIT_MPI', "1")  # Always Init MPI (support python)
-
-	# TODO: This is very fragile. The vars only exist if we compiled.
-        if hasattr(self, '_incflags'):
-            run_env.set('ND_INCFLAGS', self._incflags)
-            run_env.set('ND_LOADFLAGS', self._loadflags)
+        # run_env.set('NEURON_INIT_MPI', "1")  # Always Init MPI (support python)
```

Comment:
Why is this commented out? Could we add a comment to that?

Reply:
Makes sense. I'll remove and add a comment in the PR